modules/tablemetadata/model/tablemetadata.class.php

```diff
@@ -493,8 +516,8 @@ private function update_db($args) {
                     }
                     else
                     {
-                        $_lib['message']->add("Finnes ikke type: $type");
-                        $_lib['message']->add("table:" .  $table ." type:".$type." type_num:".$type_num." field:".$field." length: ".$len." flags:".$flags);
+                        // $_lib['message']->add("Type not found: $type");
```

Comment:
Commented out just so it does not dispaly warnings on the update function for each db and clutter the output with a lot of unnecessary warnings(not necessary since the form builder helper is not implemented fully).